server/src/internalClusterTest/java/org/opensearch/remotestore/RemoteStoreBaseIntegTestCase.java

```diff
@@ -153,34 +165,64 @@ protected BulkResponse indexBulk(String indexName, int numDocs) {
         return client().bulk(bulkRequest).actionGet();
     }
 
-    public static Settings remoteStoreClusterSettings(String segmentRepoName) {
-        return remoteStoreClusterSettings(segmentRepoName, segmentRepoName);
+    public static Settings remoteStoreClusterSettings(String name, Path path) {
+        return remoteStoreClusterSettings(name, path, name, path);
     }
 
     public static Settings remoteStoreClusterSettings(
         String segmentRepoName,
+        Path segmentRepoPath,
         String translogRepoName,
-        boolean randomizeSameRepoForRSSAndRTS
+        Path translogRepoPath
     ) {
-        return remoteStoreClusterSettings(
-            segmentRepoName,
-            randomizeSameRepoForRSSAndRTS ? (randomBoolean() ? translogRepoName : segmentRepoName) : translogRepoName
-        );
-    }
-
-    public static Settings remoteStoreClusterSettings(String segmentRepoName, String translogRepoName) {
-        Settings.Builder settingsBuilder = Settings.builder()
-            .put(CLUSTER_REMOTE_STORE_ENABLED_SETTING.getKey(), true)
-            .put(CLUSTER_REMOTE_SEGMENT_STORE_REPOSITORY_SETTING.getKey(), segmentRepoName)
-            .put(CLUSTER_REMOTE_TRANSLOG_REPOSITORY_SETTING.getKey(), translogRepoName);
+        Settings.Builder settingsBuilder = Settings.builder();
 
         if (randomBoolean()) {
             settingsBuilder.put(CLUSTER_REPLICATION_TYPE_SETTING.getKey(), ReplicationType.SEGMENT);
         }
 
+        settingsBuilder.put(buildRemoteStoreNodeAttributes(segmentRepoName, segmentRepoPath, translogRepoName, translogRepoPath));
+
         return settingsBuilder.build();
     }
 
+    public static Settings buildRemoteStoreNodeAttributes(
```

Comment:
Should we assert in the base class during setup and tear down that all nodes in the cluster were consistent?